cli/pcluster/cfnconfig.py

```diff
@@ -578,6 +581,17 @@ def __init_tags(self):
                     self.tags[key] = self.args.tags[key]
         except AttributeError:
             pass
+    
+    def __init_notificationARNs(self):
+        """
+        Get notification ARNs for cluster's event notification service (SNS)
+        """
+        self.notificationARNs = {}
```

Comment:
A dict is not a good default for notificationARNs as the CloudFormation API expects a list. In the case that notificationARNs is not included in the config, this will cause an error (as it did in my testing). To fix, please change to `self.notificationARNs = []`